enterprise_access/apps/bffs/response_builder.py

```diff
@@ -121,9 +121,6 @@ def build(self):
             'enterprise_course_enrollments': self.enterprise_course_enrollments,
         })
 
-        # Add any errors and warnings to the response
-        self.add_errors_warnings_to_response()
```

Comment:
[inform] `self.add_errors_warnings_to_response()` is called by `super().build()` above; it only needs to be re-called after adding an error/warning, which is only done below.